Content.Shared/Damage/DamageClass.cs

```diff
@@ -24,7 +25,7 @@ public static ImmutableList<DamageType> ToTypes(this DamageClass @class)
             return DamageSystem.ClassToType[@class];
         }
 
-        public static Dictionary<DamageClass, T> ToNewDictionary<T>()
+        public static Dictionary<DamageClass, T?> ToNewDictionary<T>()
```

Comment:
Paging @DrSmugleaf

Reply:
`where T : struct` instead

Reply:
Changed to `public static Dictionary<DamageClass, T> ToNewDictionary<T>() where T : struct` as Smug suggested.